src/test/java/core/basesyntax/basesyntax/strategy/PurchaseTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.basesyntax.strategy;
+
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.storage.Storage;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PurchaseTest {
+    private static final String PURCHASE = "p";
+    private static final String YABLOKO = "yabloko";
+    private static final String BANANCHIK = "bananchik";
+    private static OperationHandler handler;
+    private static Transaction transactionApple;
+    private static Transaction transactionBanana;
+    private static Fruit yabloko;
+    private static Fruit bananchik;
```

Comment:
let's use english
and you can use your models from `src/main/java` - there is nothing wrong with it